share/spice/dogo_movies/subtitle_content.handlebars

```diff
@@ -0,0 +1,16 @@
+<img src="//cdn.dogomedia.com/assets/dogomovies-6959aaba3ab072822498c8e557a0c19a.png" alt="DOGOmovies"/>
```

Comment:
Let's simplify this. Please remove the custom Dogo logo and MPAA image/CSS. We can stick to plain text here and mimic the current Movies IA display:

> Runtime: 1 hr. 30 min. | PG